### PR TITLE
Feature/ui login

### PR DESCRIPTION
**Pull request by BobAlexandraDiana**

### CODE REVIEW

File: resource-sharing-ui/src/App.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react';
+import {Route, Router, Switch} from "react-router";
+import {Provider} from "react-redux";
+
+import LoginPanel from "./components/LoginPanel";
+import HomePanel from "./components/HomePanel";
+
+class App extends Component {
+
+
```
Comment (LeancaBogdan) [inline resource-sharing-ui/src/App.js]: Please remove redundant white lines

File: resource-sharing-ui/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (LeancaBogdan) [inline resource-sharing-ui/public/manifest.json]: Please change the default app name to something more specific.

File: resource-sharing-ui/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "Create React App Sample",
```
Comment (LeancaBogdan) [inline resource-sharing-ui/public/manifest.json]: Same here

File: resource-sharing-ui/public/manifest.json (hunk 2 above)
Comment (BobAlexandraDiana) [reply resource-sharing-ui/public/manifest.json]: won't do

File: resource-sharing-ui/public/manifest.json (hunk 3 above)
Comment (BobAlexandraDiana) [reply resource-sharing-ui/public/manifest.json]: won't do for now

**Pull request merged**